tests/asg/asg_test.go

```diff
@@ -21,6 +21,7 @@ import (
 
 const (
 	scaleTimeout          = 10 * time.Minute
+	defaultVstate         = 1
```

Comment:
`defaultVState` constant should not be added

Reply:
removed it